examples/table.rs

```diff
@@ -115,15 +150,22 @@ fn run_app<B: Backend>(terminal: &mut Terminal<B>, mut app: App) -> io::Result<(
 }
 
 fn ui(f: &mut Frame, app: &mut App) {
-    let rects = Layout::vertical([Constraint::Percentage(100)]).split(f.size());
+    let rects = Layout::default()
+        .direction(Direction::Vertical)
+        .constraints([Constraint::Min(5), Constraint::Length(3)])
+        .split(f.size());
+
+    let curr_color = EXAMPLE_TABLE_COLORS[app.color_index];
 
-    // colors from https://tailwindcss.com/docs/customizing-colors
-    let header_bg = Color::from_str("#1e3a8a").unwrap();
-    let header_fg = Color::from_str("#eff6ff").unwrap();
+    let buffer_bg = curr_color.c900;
+    let header_bg = curr_color.c900;
+    let header_fg = material::WHITE;
```

Comment:
I don't think the 2014 material color palettes have a particularly dark colors: compare:
https://docs.rs/ratatui/0.26.0-alpha.1/ratatui/style/palette/material/index.html
<img width="603" alt="image" src="https://github.com/ratatui-org/ratatui/assets/381361/3f29e6fb-5f47-4d64-b87d-5dd5d14dfb52">

https://docs.rs/ratatui/0.26.0-alpha.1/ratatui/style/palette/tailwind/index.html
<img width="516" alt="image" src="https://github.com/ratatui-org/ratatui/assets/381361/c47d57c7-9326-4247-9a89-e35c5561ccfe">

Reply:
Maybe we can let users change `palette::material` to `palette::tailwind` with pressing `TAB` key? Also, they can change between fully colored bg to a darker version.

Reply:
I agree that this dark background version is better for eyes, however, the real implementations might be much smaller, so the colored version might be preferred. So I think besides choosing one implementation for colors we can let users explore them with the `TAB` key.

Reply:
The examples are really about showing a specific behavior. The table example should be less about showing off tailwind / material / being able to switch schemes. and More about showing off the features of the table. I'd suggest keep it muted, and keep the color changes to the end of the gif, cycliing through with at least 1 sec and perhaps 2 secs. To avoid "jumping" when the user reaches the end of the loop, you'll either need a longer pause at the end or make sure to make it back to where the loops starts.

Reply:
First implementation (PR):

![table](https://github.com/ratatui-org/ratatui/assets/30180366/48497bc5-25a6-41f8-b5de-f451e8b5b746)

`palette::material` changed to `palette::tailwind` and has darker background https://github.com/ratatui-org/ratatui/pull/840#discussion_r1456842572:

![table](https://github.com/ratatui-org/ratatui/assets/30180366/d01343ac-63ed-4868-b263-7939d179c030)

Colors changed to `[tailwind::BLUE, tailwind::EMERALD, tailwind::INDIGO, tailwind::RED]` https://github.com/ratatui-org/ratatui/pull/840#discussion_r1456704953:

![table](https://github.com/ratatui-org/ratatui/assets/30180366/7979050b-a355-4264-97b1-398b35f6200b)

I'm not sure which one should we choose because both versions look good to me.

Reply:
Now I realize if we put Sleep 2s or 1s before going down again to the first place it will look better

Reply:
Referring to this issue: https://github.com/ratatui-org/ratatui/issues/846

 I'll push the third option

Reply:
Either is fine